src/components/IdeaInfoOverlay/index.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from 'react';
+import './IdeaInfoOverlay.css'
+
+const IdeaInfoOverlay = ({
+
+}) => {
+
+	const [open, setOpen] = useState(true);
+
+	return (
+		<div>
+			{ open
+				&& <div className='IdeaInfoOverlay'>
+					<div className="IdeaInfoOverlay-Container">
+						<div className="IdeaInfoOverlay-Box">
+							<div className="IdeaInfoOverlay-Close" onClick={() => setOpen(false)}>
+								close
+							</div>
+							<div className="IdeaInfoOverlay-Title">
+								Idea Name
+							</div>
+							<div className="IdeaInfoOverlay-Description">
+								Idea description Idea descriptionIdea descriptionIdea descriptionIdea
```

Comment:
Improvement idea: https://baconipsum.com/ 🤣

Reply:
I will fix it with next PR, when I use redux